### PR TITLE
Verify page moved tags under approve

### PR DESCRIPTION
**Pull request by ericlvogt**
Moved tags under approve there should be no confusion now.
image

### CODE REVIEW

**Comment by dadiorchen** [comment]
@Davidezrajay Do you think this change is reasonable?  Comparing what we have currently:

image

Move the tag and species port into the section of 'approve'.

**Comment by ericlvogt** [comment]
Forgot to link the issue before. This PR addresses issue #300 to prevent confusion of what is saved on submit. On reject submit only the reject reason is saved therefore all other tags have been moved under approve.

**Comment by dadiorchen** [comment]
@Davidezrajay what's you thoughts on this?

File: client/src/components/TreeImageScrubber.js [hunk 1]
```diff
@@ -654,34 +606,84 @@ function SidePanel(props){
...
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('palm')}
-                value='palm' control={<Radio/>} label='Palm' />
-              <FormControlLabel 
-                onClick={() => handleCaptureApprovalTag('timber')}
-                value='timber' control={<Radio/>} label='Timber' />
-            </RadioGroup>
+            <Grid container direction={'column'} className={classes.sidePanelContainer}>
+              <Grid>
+                <Typography variant='h6' >Tags</Typography>
```
Comment (nmcharlton) [inline client/src/components/TreeImageScrubber.js:611]: I think the __Tags__ title should be above the tabs as it applies to both approval and rejection

**Review by nmcharlton — DISMISSED** [review]
Sorry for the long delay in reviewing.

File: client/src/components/TreeImageScrubber.js (hunk 1 above)
Comment (nmcharlton) [reply client/src/components/TreeImageScrubber.js:611]: Scrap that - it's fine as it is.

**Comment by TANguyen1893** [comment]
Any further development should be done in the admin panel client repo: https://github.com/Greenstand/treetracker-admin-client

**Pull request closed without merging**